runtime/android/core_internal/BUILD.gn

```diff
@@ -0,0 +1,215 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//build/config/android/rules.gni")
+import("//xwalk/build/version.gni")
+import("//xwalk/build/xwalk.gni")
+
+android_library("xwalk_core_internal_java") {
+  deps = [
+    ":xwalk_core_internal_java_resources",
+    "//base:base_java",
+    "//components/navigation_interception/android:navigation_interception_java",
+    "//components/web_contents_delegate_android:web_contents_delegate_android_java",
+    "//content/public/android:content_java",
+    "//media/base/android:media_java",
+    "//net/android:net_java",
+    "//ui/android:ui_java",
+    "//xwalk/extensions/android:xwalk_core_extensions_java",
+  ]
+  srcjar_deps = [ ":bridge" ]
```

Comment:
Once crosswalk-project/chromium-crosswalk#384 lands, you should be able to get rid of the `bridge` target altogether and just add a dependency on `xwalk_core_reflection_layer_java_gen` and add the path to `bridge.srcjar` as `srcjars` instead of `srcjar_deps`.